projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -75,3 +99,18 @@ export const enum TitlePosition {
   Header = 'header',
   Footer = 'footer'
 }
+
+export const enum TitleStyle {
+  Regular = 'regular',
+  GrayedOut = 'grayed-out'
+}
+
+export const enum LinkButtonStyle {
+  Primary = 'primary',
+  Quaternary = 'quaternary'
+}
+
+export const enum HeaderPosition {
```

Comment:
Since these values are representing justification of content, it should be renamed to `TitledContentHeaderJustify`

Reply:
Renamed